src/Iteration.cpp

```diff
@@ -174,6 +175,24 @@ Iteration::close( bool _flush )
     return *this;
 }
 
+Iteration &
+Iteration::open()
+{
+    Series * s = &auxiliary::deref_dynamic_cast< Series >(
+        parent->attributable->parent->attributable );
+    // figure out my iteration number
+    auto begin = s->indexOf( *this );
+    //auto end = begin;
+    //++end;
+    s->openIteration( begin->first, *this );
+    //s->flush_impl( begin, end ); // does not work as expected
+    //s->flush(); // does not work as expected
+    this->flush();
+    IOHandler->flush();
```

Comment:
```suggestion
    // set dirty, so Series::flush will open the file
    this->dirty() = true;
    s->flush();
    this->dirty() = false;
```

`Series::openIteration` is for opening an iteration's file in file-based layout, so that will likely break if using group-based iteration layout.

Reply:
Oh, thanks!
Maybe we need to note that in the doxygen string of `openIteration`